forge-gui/res/cardsfolder/upcoming/blood_for_the_blood_god.txt

```diff
@@ -0,0 +1,12 @@
+Name:Blood for the Blood God!
+ManaCost:8 B B R
+Types:Instant
+S:Mode$ ReduceCost | ValidCard$ Card.Self | Type$ Spell | Amount$ X | EffectZone$ All | Description$ This spell costs {1} less to cast for each creature that died this turn.
+A:SP$ Discard | Cost$ 8 B B R | Mode$ Hand | Defined$ You | SubAbility$ DBDraw | SpellDescription$ Discard your hand, then draw eight cards.
+SVar:DBDraw:DB$ Draw | Defined$ You | NumCards$ 8 | SubAbility$ DBKhorne
+SVar:DBKhorne:DB$ DealDamage | Defined$ Player.Opponent | NumDmg$ 8 | SubAbility$ DBChangeZone
+SVar:DBChangeZone:DB$ ChangeZone | Origin$ Stack | Destination$ Exile
+AI:RemoveDeck:Random
+AI:RemoveDeck:All
```

Comment:
I think the AI should be able to play this, maybe just a `NeedsToPlayVar` check regarding hand size?
@Agetian